example-code/snippets/src/backend/SchemaXmlFileLocater.test.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { SchemaContext, SchemaKey, SchemaMatchType } from "@itwin/ecschema-metadata";
+import { SchemaXmlFileLocater } from "@itwin/ecschema-locaters";
+import { KnownLocations } from "@itwin/core-backend";
+import path from "path";
+import { assert } from "chai";
+import * as fs from "fs-extra";
+
+describe("SchemaXmlFileLocater - locate standard schema", () => {
+  it("Schema path is less than 260 character long", async () => {
+    const cont = new SchemaContext();
+    const loc = new SchemaXmlFileLocater();
+    cont.addLocater(loc);
+    const schemaPath: string = path.join(KnownLocations.nativeAssetsDir, "ECSchemas", "Standard");
+    loc.addSchemaSearchPath(schemaPath);
+
+    assert.isTrue(schemaPath.length < 260);
+    const schemaKey = new SchemaKey("Units", 1, 0, 0);
+    const schema = await cont.getSchema(schemaKey, SchemaMatchType.Latest);
+
+    assert.isDefined(schema);
+    assert.strictEqual(schema?.name, "Units");
+  });
+
+  it("Schema path is between 260 and 1024 character long", async () => {
```

Comment:
What happens if path is > 1024 characters? Is it documented somewhere?

Reply:
According to this thread https://discussions.apple.com/thread/250275651?sortBy=best, the maximum file path length on macOS is 1024 characters, while Linux allows for paths up to 4096 characters.